### DISCUSSION TITLE
Give claim exploit/feature request

### DISCUSSION THREAD

**Discussion by feroxmc** [GriefPrevention/GriefPrevention#650] [Ideas]
/Giveclaim can be exploited as it does not even inform the recipient who gave the claim and does not ask for permission. This can be used to decrease someone's total claim block total and place them randomly throughout the world meaning they have to unclaim all to get those claimblocks back.
How to replicate: Make a claim, give it to someone. Even if their total is lower, they can give many small claims.

**Comment by Jikoo** [comment]
Claim transfer is an admin ability and is not designed for general player use.

Based on a quick read of the code I suspect that regular players with the correct permissions can also abuse `/transferclaim` to create admin claims.
